### PR TITLE
feat(auth): add `Edit Credentials` command

### PR DESCRIPTION
**Pull request by JadenSimon**
## Problem
The 'Create Credential Profile' command skips the 'create profile' wizard if any credential files already exist. This is confusing as there is no other command to explicitly edit the file(s).

## Solution
Add `Edit Credentials` command which:
* Generates a credentials file if it doesn't already exist
* Opens the file(s) 
* Prompts the user to create a new profile if none are present

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -144,13 +144,14 @@ export async function activate(context: vscode.ExtensionContext) {
...
             ),
-            Commands.register('aws.credential.profile.create', async () => {
+            Commands.register('aws.credentials.profile.create', async () => {
                 try {
                     await globals.awsContextCommands.onCommandCreateCredentialsProfile()
                 } finally {
                     telemetry.recordAwsCreateCredentials()
                 }
             }),
+            Commands.register('aws.credentials.edit', () => globals.awsContextCommands.onCommandEditCredentials()),
```
Comment (jpinkney-aws) [inline src/extension.ts:154]: Is it worth it to try and get telemetry on this to understand how many people are editing credentials?

Comment (JadenSimon) [reply src/extension.ts:154]: Not really due to https://github.com/aws/aws-toolkit-vscode/pull/2650.
https://github.com/aws/aws-toolkit-vscode/pull/2745 will make this instrumentation (almost) free though.

File: package.json [hunk 2]
```diff
@@ -1752,8 +1757,19 @@
                 }
             },
             {
-                "command": "aws.credential.profile.create",
-                "title": "%AWS.command.credential.profile.create%",
+                "command": "aws.credentials.profile.create",
+                "title": "%AWS.command.credentials.profile.create%",
+                "category": "%AWS.title%",
+                "enablement": "!isCloud9",
```
Comment (justinmk3) [inline package.json:1763]: Does this prevent the command showing in the command palette? Let's enable it in Cloud9 to start with.

Comment (JadenSimon) [reply package.json:1763]: No, I think Cloud9 hasn't implemented "enablement". But I'll remove it anyway.

**Review by justinmk3 — APPROVED** [review]

**Review by jpinkney-aws — APPROVED** [review]

**Pull request merged**